suite-native/intl/src/en.ts

```diff
@@ -393,6 +393,10 @@ export const en = {
         accountDetail: {
             accountLabelBadge: 'Run on {accountLabel}',
         },
+        emptyState: {
+            title: 'No assets',
+            subtitle: 'Connect your Trezor or sync coins to view and track assets.',
```

Comment:
So now if you search accounts through the search bar and nothing is found, there will be displayed message `Connect your Trezor or sync coins to view and track assets.`? 

Does not make sense to me. @mnuky  Are you aware of that?

Reply:
this message should only appears in empty state, not when user is actually searching